docs/tutorial.md

```diff
@@ -321,25 +314,23 @@ git-source-skaffold-git-tck6k
 image-digest-exporter-hlbsq
 ```
 
-The status of type `Succeeded` shows the Task ran successfully and you
-can also validate the Docker image is created in the location specified in the
-resource definition.
+The `Succeeded` status indicates the `Task` has completed with no errors. You
+can also confirm that the output Docker image has been created in the location specified in the resource definition.
 
-If you run into issues, use the following command to receive the logs:
+To view detailed information about the execution of your `TaskRun`, view the logs:
 
 ```bash
 tkn taskrun logs build-docker-image-from-git-source-task-run
 ```
 
-## Pipeline
+## Creating and running a `Pipeline`
 
-A [`Pipeline`](pipelines.md) defines a list of `Tasks` to execute in order, while
-also indicating if any outputs should be used as inputs of a following `Task` by
-using [the `from` field](pipelines.md#from) and also indicating
-[the order of executing (using the `runAfter` and `from` fields)](pipelines.md#ordering).
-The same variable substitution you used in `Tasks` is also available in a `Pipeline`.
+A [`Pipeline`](pipelines.md) defines an ordered series of `Tasks` that you want to execute
+along with the along with corresponding inputs and outputs for each `Task`. You can specify whether the output of one
```

Comment:
typo - along with the along with

Reply:
Fixed in https://github.com/tektoncd/pipeline/pull/2213.